cmd/gossamer/flags.go

```diff
@@ -232,6 +232,11 @@ var (
 		Name:  "pubdns",
 		Usage: "Overrides public DNS used for peer to peer networking",
 	}
+	// NodeKeyFlag uses the supplied hex-encoded Ed25519 secret key seed for libp2p networking
+	NodeKeyFlag = cli.StringFlag{
+		Name:  "node-key",
+		Usage: "Overrides the secret key to use for libp2p",
```

Comment:
Should we precise the `ed25519` somewhere in the variable name/flag and at least in its usage?